recipes/react-router/app/entry.server.tsx

```diff
@@ -0,0 +1,68 @@
+import { PassThrough } from "node:stream";
```

Comment:
Is this the "revealed" file? I'd actually rather not include this since that's not the default behaviour